GoogleForCreators/web-stories-wp
# Video Optimization: Settings Field

## Context



## Summary

Add a new checkbox for opting out of video optimization. 

## Relevant Technical Choices

Much of the code for this is copied from the existing tracking opt in. 

## To-do



## User-facing changes

Users will be able to opt out of video optimization via the setting screen in the dashboard. 

## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Enable video optimization in experiences. 
2. Navigation to Web stories > settings. 
3. Check / uncheck the video optimization check box. 
4. Refresh page / navigation away and come back, see setting remain as selected.  

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #5981
